beacon-chain/sync/validate_beacon_attestation.go

```diff
@@ -71,14 +70,6 @@ func (s *Service) validateCommitteeIndexBeaconAttestation(ctx context.Context, p
 	if data.Slot == 0 {
 		return pubsub.ValidationIgnore, nil
 	}
-	// Broadcast the unaggregated attestation on a feed to notify other services in the beacon node
-	// of a received unaggregated attestation.
-	s.cfg.attestationNotifier.OperationFeed().Send(&feed.Event{
-		Type: operation.UnaggregatedAttReceived,
-		Data: &operation.UnAggregatedAttReceivedData{
-			Attestation: att,
-		},
-	})
 
```

Comment:
Are there consumers of this feed? is this a breaking change? If so we may want to keep the deneb path unaltered by broadcasting if the version is pre-Elecrta

Reply:
Ahh nevermind this broadcast has moved to later.